plugins/inventory/scaleway.py

```diff
@@ -30,9 +32,9 @@
             description: Filter results on a specific tag.
             type: list
         oauth_token:
-            required: True
             description:
             - Scaleway OAuth token.
+            - If not explicitly defined or in environment variables, it will try to lookup in the scaleway-cli configuration file.
```

Comment:
I'd include the filename for clarity:
```suggestion
            - If not explicitly defined or in environment variables, it will try to lookup in the scaleway-cli configuration file
              (C(~/.config/scw/config.yaml))).
```